packages/react-checkbox/src/components/Checkbox/renderCheckbox.tsx

```diff
@@ -1,19 +1,19 @@
 import * as React from 'react';
-import { getSlotsCompat } from '@fluentui/react-utilities';
-import { CheckboxState } from './Checkbox.types';
+import { getSlots } from '@fluentui/react-utilities';
+import { CheckboxState, CheckboxSlots } from './Checkbox.types';
 import { checkboxShorthandProps } from './useCheckbox';
 
 export const renderCheckbox = (state: CheckboxState) => {
-  const { slots, slotProps } = getSlotsCompat(state, checkboxShorthandProps);
+  const { slots, slotProps } = getSlots<CheckboxSlots>(state, checkboxShorthandProps);
 
   return (
     <slots.root {...slotProps.root}>
-      {state.labelPosition === 'before' && state.children}
-      <div className={state.containerClassName}>
+      {state.labelPosition === 'before' && slotProps.root.children}
+      <slot className={state.containerClassName}>
```

Comment:
`slot`?

Reply:
I guess that this should just a `div` for now, but we should follow up to have this as a slot...